src/interface/sparse_pack.hpp

```diff
@@ -443,6 +443,12 @@ class SparsePack : public SparsePackBase {
   // and it may not even be an lvalue.
   static std::vector<MetadataFlag> AddFlag_(const std::vector<MetadataFlag> &flags,
                                             MetadataFlag mf = Metadata::WithFluxes) {
+    // TODO(LFR): We need to decide if variables w/o fluxes should end up a pack that asks
+    //            for WithFluxes. I actually use this reasonably often in practice, even
+    //            though I was the one who suggested enforcing that WithFluxes
+    //            PackDescriptors include the Metadata::WithFluxes flag.
```

Comment:
I don't feel strongly either way. If you're using that, just get rid of this functionality.

Reply:
That is what I think as well. After merging your changes, forcing `WithFluxes` variables to have `Metadata::WithFluxes` caused a bug in my code (since the sparse packs I was creating had some variables that had fluxes and others that didn't). It just quietly removes variables that you ask for that don't have fluxes, which inevitably causes segfaults.

Reply:
Ok, just removed it. Sorry for suggesting that in the first place.